CONTRIBUTING.md

```diff
@@ -94,3 +108,13 @@ Mixxx core team members are contributors who have write access to the [upstream
 * Merge PRs using a merge, to keep the original commits valid. Keep the default commit message "Merge pull request ..." with the reference to the pull request. In case where the PR contains broken (non-building) commits, back-and-forth commits or commits without a meaningful commit message that are not worth keeping, ask the author to squash the commits before merge. Alternatively you may ask the contributor to check "Allow edits and access to secrets by maintainers". Then you can squash locally or use the `/softfix` comment to squash remotely. See [Softfix](https://github.com/daschuer/softfix/?tab=readme-ov-file#softfix-a-pull-request)
 * Default to open; only post in the private Zulip stream for discussions that have a reason to be private. Most of the time, post to a public Zulip stream so anyone can participate in the discussion.
 * When Mixxx participates in Google Summer of Code, you may volunteer as a mentor if you like.
+
+#### Change requests
+
+We recommend that core members use the `Change request` review state carefully due to the blocking aspect it represent in PRs. When a member submit a change request, they are expected to commit to provide quick feedback once the PR's author has addressed these changes in order not to block the PR. If a core member request changes and doesn't provide a feedback within 15 days following the author actor acting on the this request, another core member can discard their review in order to unblock the PR. We trust the second core member to ensure that the initial change request has correctly been addressed. When in doubt, the initial change request should be kept. Note that the [CODEOWNERS file](.github/CODEOWNERS) can be used to find another member with expertise to help assessing whether or not the initial concerned has been addressed.
```

Comment:
This is not the process we have discussed on Zulip: 
``` 
-     PR written
-     reviewer leaves comments
-     time passes
-     author says "hey I think I addressed your comments"
-     time passes
-     author says "hi are you still around or can we dismiss the review?"
-     reviewer either says "yes please do not dismiss" or more time passes (at least a week or two) and then we dismiss
```

Reply:
I agree, let's try to encode that process as we agreed, including the notification step and the double 1 week wait time.